docs/guides/hybrid-graphics.md

```diff
@@ -9,6 +9,17 @@ This has been tested on the MacBookPro16,1 and the MacBookPro15,1. The 15,3 and
 1. Resume after suspend is broken, as the GMUX (graphics multiplexer) doesn't connect the iGPU to the display after resuming. For this to be fixed, a Linux driver for `acpi:APP000B:GPUC:` needs to be written (macOS uses AppleMuxControl2.kext). The extra battery life may make this a worthwhile trade-off (about 3 hours to almost 6 hours on a MacBookPro16,1)
 2. If using `DRI_PRIME=1` on programs causes system crashes, with "[CPU CATERR](https://gist.github.com/Redecorating/956a672e6922e285de83fdd7d9982e5e#gistcomment-3719941)" problem reports in macOS, disable dynamic power management with the `amdgpu.dpm=0` kernel argument, or `echo high | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`.
 3. If apple-set-os is loaded, the iGPU will control display brightness, and if the iGPU isn't the boot gpu, the i915 Intel graphics driver will not load, and the display brightness cannot be changed (The exception to this is sometimes when rebooting from macOS Recovery etc, i915 loads fine).
+4. Some users with  MacBookPro16.1 and amdgpu 5500m models have reported problems. Unable to shutdown normally(sudden fan noise), getting T2 chip reset and excessive high temperatures under normal conditions. Posible workarounds are:
+    1. Set amd dynamic power management from auto to low. `echo low | sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level`
+
+        To apply the low level permanently:
```

Comment:
can you put "create `<filename>` with the following contents" at the end of this line, instead of in the code block?

Reply:
ok

Reply:
about redesigning the page, I think is better if leave that you guys.
You could do it in another revision as you think is better.